### PR TITLE
multi_nics_stress:match vm name and netperf_client vm name

### PR DESCRIPTION
**Pull request by kimi1978wy**
change ${main_vm} to vm1

Signed-off-by: Yu Wang <wyu@redhat.com>

id: 1488074

### CODE REVIEW

**Comment by kimi1978wy** [comment]
@Zhengtong could you help to review this patch?
Thanks.

**Comment by Zhengtong** [comment]
@kimi1978wy , If we add 4 nics in the first vm, I think we should do dhclient for the other 3 nics. because the other 3 nics didn't get IP in my testing.. So the netperf data stream won't get through.

File: qemu/tests/cfg/multi_nics_stress.cfg [hunk 1]
```diff
@@ -6,10 +6,10 @@
     image_snapshot = yes
     hostpassword = redhat
     # nic1 is for control, nic2 is for data connection
-    vms = "vm1 vm2 vm3 vm4 vm5"
+    vms += " vm2 vm3 vm4 vm5"
     netperf_client = ${main_vm}
     netperf_server = "vm2 vm3 vm4 vm5"
-    nics_vm1 = 'nic1 nic2 nic3 nic4'
+    nics_avocado-vt-vm1 = 'nic1 nic2 nic3 nic4'
```
Comment (hereischen) [inline qemu/tests/cfg/multi_nics_stress.cfg]: It is OK to me.
But I think it may be better to use same naming pattern for all vms.

Comment (kimi1978wy) [reply qemu/tests/cfg/multi_nics_stress.cfg]: @Zhengtong I will open another bug to follow dhclient other 3 nics problem.

**Comment by kimi1978wy** [comment]
@Zhengtong I will open another bug to follow dhclient other 3 nics problem.

**Comment by kimi1978wy** [comment]
@hereischen I change ${main_vm} to avocado-vt-vm1, in order to "default name" not influence the script.  And  I did not change the first vm name, since it is the only client for this test , it is not the same as other vms.

**Comment by Zhengtong** [comment]
Acked-by: Zhengtong Liu <zhengtli@redhat.com>

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Comment by kimi1978wy** [comment]
According to xuhan's advice, change ${main_vm} to vm1, since the auto default vm name will not influence this case.

**Pull request merged**